finance/process/request-extension.md

```diff
@@ -0,0 +1,16 @@
+# Guidelines for Requesting a Project Extension
+
+In general, projects financially supported by Astropy are subject to contracts or other agreements with a specified period of performance.
+Under normal circumstances, we expect the work to be completed within that period.
+However, we recognize that this is occasionally impossible and that an extension may be necessary.
+
+In order to request an extension to an approved proposal, the entity (person or organization) responsible for carrying out the work should open a new pull request to update the approved proposal as stored in the [astropy-project repository](https://github.com/astropy/astropy-project).
+The PR should add a new section to the existing proposal text which specifies:
+
+- What work will be undertaken during the period covered by the extension.
+  In particular, please be sure to note if the full scope of the original proposal is *not* expected to be completed during the extension.
+- The funds that will be spent during the extension.
+- The new end date for the project.
+
```

Comment:
```suggestion
- The PR description should give a brief explanation for the reason of the delay. (If this involves confidential matters, it can instead say "the reasons have been communicated to the Astropy Finance Committee" or similar.)

```